src/sections/sponsorship/Sponsorship.Info.tsx

```diff
@@ -1,17 +1,29 @@
-import React from "react"
+import React, { useEffect } from "react"
 import "./styles.less"
 
+import Arrow from "../../assets/arrow.svg"
 import SponsorHeroLogo from "../../assets/sponsor-images/sponsor_hero.png"
 
 const SponsorshipInfo: React.FC = () => {
+  const [scrolled, setScrolled] = React.useState(false)
+
+  useEffect(() => {
+    if (!scrolled) {
+      const handleScroll = () => {
+        setScrolled(true)
+        window.removeEventListener("scroll", handleScroll)
+        return undefined
+      }
+
+      window.addEventListener("scroll", handleScroll)
+      return () => window.removeEventListener("scroll", handleScroll)
+    }
+    return undefined
+  }, [scrolled])
+
```

Comment:
This might be a nice consideration for later, but I'm noticing this dropdown arrow is reused in a few places; could probably keep things DRY and refactor this into a component by itself.